product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx

```diff
@@ -6,9 +6,8 @@ navigation:
 ---
 
 If you plan on deploying a specific version of Postgres Distributed or a specific Postgres distribution or version, you will need to select the appropriate images and image versions.
-Gather the following information before [installing EDB Postgres Distributed for Kubernetes](/postgres_distributed_for_kubernetes/latest/installation_upgrade):
```

Comment:
"Gather information" doesn't properly introduce the list, since each of the items is a complete action to perform.